themes/dark/dracula.css

```
@@ -0,0 +1,89 @@
+/* [THEME META] {"ver":1, "author":"woooferz", "description":"Dracula Theme (pretty cool)", "accessibility":"optimized"} */
```

> Accessibility optimized themes are reviewed to a higher standard than the default accessibility level. They should be free of all accessibility issues (up to AAA) on all states (hover, active, etc).

This theme does not meet the requirements for optimized accessibility. Please check all your contrast ratios towards [the WebAIM guidelines](https://webaim.org/resources/contrastchecker/), or opt-out back to standard accessibility.

```suggestion
/* [THEME META] {"ver":1, "author":"woooferz", "description":"Dracula Theme (pretty cool)"} */
```

ℹ️ A standard accessibility test was not performed due to issues detected with optimized accessibility.